modular_zubbers/code/modules/alternative_job_titles/code/alt_job_titles.dm

```diff
@@ -1,3 +1,249 @@
+/datum/job/ai/New()
+	.=..()
+	alt_titles += list(
+		"Abominable Intelligence",
+		"Artificial Intelligence",
+		"Big Brother",
+		"Big Sibling",
+		"Big Sister",
+		"Machine Spirit",
+	)
+
+/datum/job/assistant/New()
+	.=..()
+	alt_titles += list(
+		"Bastard",
+		"Call Boy",
+		"Call Girl",
+		"Callperson",
+		"Crew Gremlin",
+		"Escort",
+		"Fiend",
+		"Fleshlight",
+		"Hooker",
+		"Jackanape",
+		"Maintworker",
+		"Ne'er-do-well",
+		"Prostitute",
+		"Roustabout",
+		"Sexworker",
+		"Slut",
+		"Spacefarer",
+		"Stripper",
+		"Troublemaker",
+		"Whore",
+	)
+//Finally, a name to represent the average Assistant.
+
+/datum/job/atmospheric_technician/New()
+	.=..()
+	alt_titles += list(
+		"Pipe Cleaner",
+		"Pipe Inspector",
+		"Pipe Technician",
+	)
+
+/datum/job/bitrunner/New()
+	.=..()
+	alt_titles += list(
+		"Netrunner",
+		"Matrix Traversal Technician",
+	)
+
+/datum/job/blueshield/New()
+	.=..()
+	alt_titles += list(
+		"Blueshit",
+		"Chief Steward",
+		"Command Enforcer",
+		"Command Goon",
+		"Command Henchman",
+		"Steward",
+	)
+
+/datum/job/botanist/New()
+	.=..()
+	alt_titles += list(
+		"Apiarist",
+		"Apiculturist",
+		"Beekeeper",
+		"Beemaster",
+		"Farmer",
+		"Sodbuster",
+	)
+
+/datum/job/bouncer/New()
+	.=..()
+	alt_titles += list(
+		"Hired Muscle",
+	)
+
+/datum/job/captain/New()
+	.=..()
+	alt_titles += list(
+		"Cap'n",
+		"Chief Mate",
+		"Offworld Installation Manager",
+		"Premier",
+		"Staff Captain",
+		"Station Admiral",
+	)
+
+/datum/job/cargo_technician/New()
+	.=..()
+	alt_titles += list(
+		"Cargo Technician",
+		"Deckhand",
+		"Deck Rating",
+	)
+
+/datum/job/chaplain/New()
+	.=..()
+	alt_titles += list(
+		"Moffpriest",
+	)
+
+/datum/job/chemist/New()
+	.=..()
+	alt_titles += list(
+		"Alchemist",
+		"Explosives Specialist",
+		"Harmacist",
+		"Harmacologist",
+		"Plumbing Technician",
+		"Plumber",
+		"Pyrotechnics Specialist",
+		"Stimulant Manufacturing Technician",
+	)
+
+/datum/job/chief_medical_officer/New()
+	.=..()
+	alt_titles += list(
+		"Chief Boo-Boo Kisser",
+	)
+
+/datum/job/cook/New()
+	.=..()
+	alt_titles += list(
+		"Baker",
+		"Confectionist",
+		"Pastry Chef",
+		"Pâtissier",
+	)
+
+/datum/job/cyborg/New()
+	.=..()
+	alt_titles += list(
+		"Corporate Appliance",
+		"Droid",
+		"Gyndroid",
+		"Stationbound",
+		"Toaster",
+	)
+
+/datum/job/doctor/New()
+	.=..()
+	alt_titles += list(
+		"Boo-Boo Repair Specialist",
+		"Magos",
+	)
+
+/datum/job/head_of_personnel/New()
+	.=..()
+	alt_titles += list(
+		"Captain't",
+		"Chief Navigational Officer",
+		"First Officer",
+		"First Mate",
+		"Second Officer",
+		"Second Mate",
+		"Staff Manager",
+		"Staff Overseer",
+	)
```

Comment:
what.

Reply:
These are all naval titles for the Head of Personnel, and a Captain't title as a silly title, what exactly is the problem here?

Reply:
I took out Captain't, the rest should be valid